documentation/homework/week4/src/stores/todo_actions.js

```diff
@@ -0,0 +1,150 @@
+import { observable, action, runInAction } from 'mobx'
+
+// Setting our localStorage
+// import { locals } from '../utils'
+
+// set the default state of the items Object
+const default_item_state = {
+  description: '',
+  deadline: '',
+}
+
+async function request(url, method, inputToString) {
```

Comment:
This is a really nice abstraction. But I'd rather see it in your `utils` directory, not inside the store. Imagine we had several stores, each one would want to use `request`.